### ISSUE TITLE
Noetic build failure catkin::pcl_conversions::wrapped-linker-option0 target not found

### ISSUE THREAD

**Issue by sloretz** [mrpt-ros-pkg/pose_cov_ops#9]
It looks like the latest release of this package started failing on Debian Buster on May 10th.

https://build.ros.org/job/Nbin_db_dB64__pose_cov_ops__debian_buster_amd64__binary/14/console

```
23:14:15 CMake Error at CMakeLists.txt:67 (add_library):
23:14:15   Target "pose_cov_ops" links to target
23:14:15   "catkin::pcl_conversions::wrapped-linker-option0" but the target was not
23:14:15   found.  Perhaps a find_package() call is missing for an IMPORTED target, or
23:14:15   an ALIAS target is missing?
23:14:15 
23:14:15 
23:14:15 CMake Error at /opt/ros/noetic/share/catkin/cmake/test/gtest.cmake:180 (add_executable):
23:14:15   Target "pose_cov_ops-test" links to target
23:14:15   "catkin::pcl_conversions::wrapped-linker-option0" but the target was not
23:14:15   found.  Perhaps a find_package() call is missing for an IMPORTED target, or
23:14:15   an ALIAS target is missing?
23:14:15 Call Stack (most recent call first):
23:14:15   /opt/ros/noetic/share/catkin/cmake/test/gtest.cmake:89 (_catkin_add_executable_with_google_test)
23:14:15   /opt/ros/noetic/share/catkin/cmake/test/gtest.cmake:37 (_catkin_add_google_test)
23:14:15   CMakeLists.txt:143 (catkin_add_gtest)
```

**Comment by jlblancoc** [comment]
Thanks for reporting. 
So far, I tracked this back to a cmake exported target in mrpt2 package, library mrpt::ros1bridge. 
For some reason, `catkin::pcl_conversions::wrapped-linker-option0` is returned by cmake as a public dependency of pcl_conversions. 

However, the pkgconfig.pc file for pcl-conversions (from [here](http://repositories.ros.org/ubuntu/building/pool/main/r/ros-noetic-pcl-conversions/)) does not have such a phony library listed in it. 

Catkin seems to generate such "fake" names intentionally: 

https://github.com/ros/catkin/blob/7fa3eb3508ba12c34d85b8e54bbdf4bbdc60a5c1/cmake/templates/pkgConfig.cmake.in#L132-L137

So I can't clearly identify when things go wrong after that... 

BTW: Everything works fine for Ubuntu, it's only in Debian builds where this happens...

Perhaps @dirk-thomas or someone else with a deep understanding of the inner workings of catkin can give me a hint of what to investigate next? Thanks.

**Comment by sloretz** [comment]
IIRC catkin creates the dummy targets to hold linker options. It's probably a target for `-L${prefix}/lib` option from the pkg-config file.

<details><summary>pcl_conversions.pc</summary>

```
prefix=/opt/ros/noetic

Name: pcl_conversions
Description: Description of pcl_conversions
Version: 1.7.4
Cflags: -I${prefix}/include -I/usr/include/eigen3 -I/usr/include/pcl-1.9 -I/usr/include -I/usr/include/vtk-7.1 -I/usr/include/freetype2 -I/usr/include/x86_64-linux-gnu -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi -I/usr/lib/x86_64-linux-gnu/openmpi/include -I/usr/include/python3.7m -I/usr/include/hdf5/openmpi -I/usr/include/jsoncpp -I/usr/include/libxml2 -I/usr/include/tcl
Libs: -L${prefix}/lib /usr/lib/x86_64-linux-gnu/libpcl_common.so /usr/lib/x86_64-linux-gnu/libpcl_octree.so /usr/lib/x86_64-linux-gnu/libpcl_io.so /usr/lib/x86_64-linux-gnu/libboost_system.so /usr/lib/x86_64-linux-gnu/libboost_filesystem.so /usr/lib/x86_64-linux-gnu/libboost_thread.so -pthread /usr/lib/x86_64-linux-gnu/libboost_date_time.so /usr/lib/x86_64-linux-gnu/libboost_iostreams.so /usr/lib/x86_64-linux-gnu/libboost_serialization.so /usr/lib/x86_64-linux-gnu/libboost_chrono.so /usr/lib/x86_64-linux-gnu/libboost_atomic.so /usr/lib/x86_64-linux-gnu/libboost_regex.so /usr/lib/x86_64-linux-gnu/libvtkChartsCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkCommonColor-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkCommonCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtksys-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkCommonDataModel-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkCommonMath-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkCommonMisc-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkCommonSystem-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkCommonTransforms-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkCommonExecutionModel-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersGeneral-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkCommonComputationalGeometry-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkInfovisCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersExtraction-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersStatistics-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingFourier-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkalglib-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingContext2D-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersGeometry-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersSources-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingFreeType-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libfreetype.so /usr/lib/x86_64-linux-gnu/libz.so /usr/lib/x86_64-linux-gnu/libvtkDICOMParser-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkDomainsChemistry-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOLegacy-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOXMLParser-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libexpat.so /usr/lib/x86_64-linux-gnu/libvtkDomainsChemistryOpenGL2-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingOpenGL2-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOImage-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkmetaio-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libjpeg.so /usr/lib/x86_64-linux-gnu/libpng.so /usr/lib/x86_64-linux-gnu/libtiff.so /usr/lib/x86_64-linux-gnu/libvtkFiltersAMR-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOXML-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkParallelCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersFlowPaths-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersGeneric-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersHybrid-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingSources-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersHyperTree-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersImaging-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingGeneral-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersModeling-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersParallel-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersParallelDIY2-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkParallelMPI-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersParallelFlowPaths-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersParallelGeometry-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersParallelImaging-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersParallelMPI-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersParallelStatistics-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersPoints-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersProgrammable-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersPython-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libpython3.7m.so /usr/lib/x86_64-linux-gnu/libvtkWrappingPython37Core-7.1.so.7.1.1 /usr/lib/libvtkWrappingTools-7.1.a /usr/lib/x86_64-linux-gnu/libvtkFiltersReebGraph-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersSMP-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersSelection-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersTexture-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkFiltersVerdict-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkverdict-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkGUISupportQt-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkInteractionStyle-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkGUISupportQtSQL-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOSQL-7.1.so.7.1.1 -lsqlite3 /usr/lib/x86_64-linux-gnu/libvtkGeovisCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkInfovisLayout-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingHybrid-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkInteractionWidgets-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingColor-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingAnnotation-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingVolume-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkViewsCore-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libproj.so /usr/lib/x86_64-linux-gnu/libvtkIOAMR-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/hdf5/openmpi/libhdf5.so /usr/lib/x86_64-linux-gnu/libsz.so /usr/lib/x86_64-linux-gnu/libdl.so /usr/lib/x86_64-linux-gnu/libm.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi.so /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi_cxx.so /usr/lib/x86_64-linux-gnu/hdf5/openmpi/libhdf5_hl.so /usr/lib/x86_64-linux-gnu/libvtkIOEnSight-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOExodus-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkexoIIc-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libnetcdf_c++.so /usr/lib/x86_64-linux-gnu/libnetcdf.so /usr/lib/x86_64-linux-gnu/libvtkIOExport-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingGL2PSOpenGL2-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libgl2ps.so /usr/lib/x86_64-linux-gnu/libvtkIOFFMPEG-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOMovie-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libtheoraenc.so /usr/lib/x86_64-linux-gnu/libtheoradec.so /usr/lib/x86_64-linux-gnu/libogg.so /usr/lib/x86_64-linux-gnu/libvtkIOGDAL-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOGeoJSON-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libjsoncpp.so /usr/lib/x86_64-linux-gnu/libvtkIOGeometry-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOImport-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOInfovis-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libxml2.so /usr/lib/x86_64-linux-gnu/libvtkIOLSDyna-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOMINC-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOMPIImage-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOMPIParallel-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOParallel-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIONetCDF-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOMySQL-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOODBC-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOPLY-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOParallelExodus-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOParallelLSDyna-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOParallelNetCDF-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOParallelXML-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOPostgreSQL-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOTecplotTable-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOVPIC-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkVPIC-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOVideo-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkIOXdmf2-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkxdmf2-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingMath-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingMorphological-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingStatistics-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkImagingStencil-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkInfovisBoostGraphAlgorithms-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkInteractionImage-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkLocalExample-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkParallelMPI4Py-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkPythonInterpreter-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingContextOpenGL2-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingExternal-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingFreeTypeFontConfig-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingImage-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingLICOpenGL2-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingLOD-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingLabel-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingMatplotlib-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingParallel-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingParallelLIC-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingSceneGraph-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingVolumeAMR-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkRenderingVolumeOpenGL2-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkTestingGenericBridge-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkTestingIOSQL-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkTestingRendering-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkViewsContext2D-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkViewsGeovis-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkViewsInfovis-7.1.so.7.1.1 /usr/lib/x86_64-linux-gnu/libvtkWrappingJava-7.1.so.7.1.1
Requires: pcl_msgs roscpp sensor_msgs std_msgs
```
</details>

Maybe there's a problem in having some things found using the `COMPONENTS` feature of `catkin`, and some things not?

Is `mrpt-ros1bridge` equivalent to `mrpt_bridge`? Maybe finding that with Catkin before is what gave catkin the opportunity to create that dummy target. https://github.com/mrpt-ros-pkg/pose_cov_ops/blob/d694fae9d207908f35442b87f9b43b656f80732d/CMakeLists.txt#L8-L12

Adding `pcl_conversions` to the `find_package(catkin ...)` here might be a way to workaround it https://github.com/mrpt-ros-pkg/pose_cov_ops/blob/2e6d6434a5bc1732b1b48b5f301853c0f9803a71/CMakeLists.txt#L20-L23

**Comment by jlblancoc** [comment]
@sloretz If this ( https://github.com/ros/rosdistro/pull/33317 ) new release of mrpt2 does not fix the issue, I'll try your suggestions. Thanks.

**Comment by jlblancoc** [comment]
@sloretz I released a new version of mrpt2 (dependency of this package) to see if it fixes this issue, but Jenkins didn't rebuild it yet ( https://build.ros.org/job/Mbin_uB64__pose_cov_ops__ubuntu_bionic_amd64__binary ) . Is it supposed to retry automatically at some point without manual intervention? Thanks

**Comment by sloretz** [comment]
`Mbin` means that's a ROS Melodic job. I see mrpt2 version 2.4.6 has built on Debian Buster for ROS Noetic.

https://build.ros.org/job/Nbin_db_dB64__mrpt2__debian_buster_amd64__binary/

pose_cov_ops built against 2.4.6, but had build failures complaining about a non-existent include directory.

https://build.ros.org/job/Nbin_db_dB64__pose_cov_ops__debian_buster_amd64__binary/25/console

<details><summary> build failures</summary>

```
23:08:59 -- Configuring done
23:08:59 CMake Error in CMakeLists.txt:
23:08:59   Imported target "mrpt::poses" includes non-existent path
23:08:59 
23:08:59     "/usr/include/suitesparse"
23:08:59 
23:08:59   in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
23:08:59 
23:08:59   * The path was deleted, renamed, or moved to another location.
23:08:59 
23:08:59   * An install or uninstall procedure did not complete successfully.
23:08:59 
23:08:59   * The installation package was faulty and references files it does not
23:08:59   provide.
23:08:59 
23:08:59 
23:08:59 
23:08:59 CMake Error in CMakeLists.txt:
23:08:59   Imported target "mrpt::poses" includes non-existent path
23:08:59 
23:08:59     "/usr/include/suitesparse"
23:08:59 
23:08:59   in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
23:08:59 
23:08:59   * The path was deleted, renamed, or moved to another location.
23:08:59 
23:08:59   * An install or uninstall procedure did not complete successfully.
23:08:59 
23:08:59   * The installation package was faulty and references files it does not
23:08:59   provide.
23:08:59 
23:08:59 
23:08:59 
23:08:59 CMake Error in CMakeLists.txt:
23:08:59   Imported target "mrpt::poses" includes non-existent path
23:08:59 
23:08:59     "/usr/include/suitesparse"
23:08:59 
23:08:59   in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
23:08:59 
23:08:59   * The path was deleted, renamed, or moved to another location.
23:08:59 
23:08:59   * An install or uninstall procedure did not complete successfully.
23:08:59 
23:08:59   * The installation package was faulty and references files it does not
23:08:59   provide.
23:08:59 
23:08:59 
23:08:59 
23:08:59 CMake Error in CMakeLists.txt:
23:08:59   Imported target "mrpt::poses" includes non-existent path
23:08:59 
23:08:59     "/usr/include/suitesparse"
23:08:59 
23:08:59   in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
23:08:59 
23:08:59   * The path was deleted, renamed, or moved to another location.
23:08:59 
23:08:59   * An install or uninstall procedure did not complete successfully.
23:08:59 
23:08:59   * The installation package was faulty and references files it does not
23:08:59   provide.
23:08:59 
23:08:59 
23:08:59 
23:08:59 -- Generating done
23:08:59 CMake Warning:
23:08:59   Manually-specified variables were not used by the project:
23:08:59 
23:08:59     CMAKE_EXPORT_NO_PACKAGE_REGISTRY
23:08:59 
23:08:59 
```
</details>

**Comment by jlblancoc** [comment]
Thanks. Yes, you are right, I noticed and fixed it a while ago via https://github.com/MRPT/mrpt/commit/147d79800d3f5ae27facea721fb9884f81a3f64a

I tried to reproduce the ros buildfarm locally using http://prerelease.ros.org/ but it seems not to work straight away, and needed fixes at each build step and still wasn't able to go through the entire process via local docker images... is the pre-release project supposed to be maintained and working? 
Any other advice on how to debug these issues, apart of making public releases and going through the real buildfarm? :-)
